ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1512,6 +1526,43 @@ export const WorkspaceEdit = fp.flow(
             {loading && (
               <SpinnerOverlay overrideStylesOverlay={styles.spinner} />
             )}
+            {showWorkspaceCreatedPopup && (
+              <Modal>
+                <ModalTitle>Workspace ready in Verily Workbench</ModalTitle>
+                <ModalBody>
+                  Your workspace "{this.state.workspace.name}" has been
+                  successfully created. Start exploring your new research
+                  environment.
+                  <br />
```

Comment:
Was this intentional? If so, adding margin might be a better way to add some spacing.